src/app/components/message/MsgTypeRenderers.tsx

```diff
@@ -88,12 +112,22 @@ export function MText({ edited, content, renderBody, renderUrlsPreview }: MTextP
         jumboEmoji={JUMBO_EMOJI_REG.test(trimmedBody)}
       >
         {renderBody({
-          body: trimmedBody,
-          customBody: typeof customBody === 'string' ? customBody : undefined,
+          body: finalContent,
+          customBody: typeof customBody === 'string' ? finalCustomContent : undefined,
         })}
         {edited && <MessageEditedContent />}
       </MessageTextBody>
       {renderUrlsPreview && urls && urls.length > 0 && renderUrlsPreview(urls)}
+
+      {shouldTruncate && (
+        <button
+          aria-expanded={isExpanded}
+          className="show-more"
```

Comment:
Please use the components from design system(Folds) and CSS-in-JS instead of global css.

Reply:
Done